docs/user/custom-resources/apigateway/apigateway-v1beta2-adr.md

```diff
@@ -0,0 +1,172 @@
+| **rules.noAuth**                 |  **NO**   | Setting `noAuth` to `true` disables authorization.                                                                                                                                                                                                                                                                                           | When `noAuth` is set to true, there cannot be any access strategy defined                                                                                                                                                        |
+| **rules.accessStrategy**         |  **NO**   | Specifies the access strategy.                                                                                                                                                                                                                                                                                                               | Supported are `extAuth` or `jwt` as [Istio](https://istio.io/latest/docs/tasks/security/authorization/authz-jwt/) access strategy.                                                                                               |
+| **rules.accessStrategy.extAuth** |  **NO**   | Specifies the list of external authorizers. For more information see the [External Authorizer ADR](https://github.com/kyma-project/api-gateway/issues/938).                                                                                                                                                                                  |                                                                                                                                                                                                                                  |
+| **rules.accessStrategy.jwt**     |  **NO**   | Specifies the Istio JWT access strategy. For more information see the [documentation](https://github.com/kyma-project/api-gateway/blob/main/docs/user/custom-resources/apirule/04-20-apirule-istio-jwt-access-strategy.md).                                                                                                                  |                                                                                                                                                                                                                                  |
```

Comment:
this has to be changed, in v1beta1 we have it as `RawExtension` handler and fields are under config. for v1beta2 this looks different. It will be nicer if we have it fully described in a similar fashion like this one in a table with all fields described.